### PR TITLE
Migrate access cluster service page

### PR DESCRIPTION
**Pull request by tengqm**
The 'acess-cluster-service' page has nothing to do with cluster administration.
A better home for this page should be the `/docs/tasks/access-application-cluster` subdirectory instead.

### CODE REVIEW

File: content/en/docs/tasks/access-application-cluster/access-cluster.md [hunk 1]
```diff
@@ -237,7 +237,9 @@ In each case, the credentials of the pod are used to communicate securely with t
 
 ## Accessing services running on the cluster
 
-The previous section describes how to connect to the Kubernetes API server. For information about connecting to other services running on a Kubernetes cluster, see [Access Cluster Services.](/docs/tasks/administer-cluster/access-cluster-services/)
+The previous section describes how to connect to the Kubernetes API server.
+For information about connecting to other services running on a Kubernetes cluster, see
+[Access Cluster Services.](/docs/tasks/access-application-cluster/access-cluster-services/)
```
Comment (sftim) [inline content/en/docs/tasks/access-application-cluster/access-cluster.md:242]: nit:
```suggestion
[Access Cluster Services](/docs/tasks/access-application-cluster/access-cluster-services/).
```

**Review by sftim — APPROVED** [review]
Thanks

/lgtm
/approve

/hold
in case you want to fix a nit here @tengqm

File: content/en/docs/tasks/access-application-cluster/access-cluster.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/tasks/access-application-cluster/access-cluster.md:242]: Fixed.

**Pull request merged**